pipelines/coref/project.yml

```diff
@@ -0,0 +1,127 @@
+title: "spaCy Coref Model"
+description: >
+  This project trains a coreference model for spaCy using OntoNotes.
+
+vars:
+  # XXX Change to your actual GPU ID
+  gpu_id: 0
+  # XXX fill with your local path to OntoNotes
+  ontonotes: /home/polm/data/ontonotes
+  max_epochs: 5
+
+directories: ["assets", "corpus", "scripts", "configs", "training"]
+
+assets:
+  - dest: "assets/"
+    git:
+      repo: "https://github.com/explosion/conll-2012"
+      branch: "main"
+      path: ""
+    description: "CoNLL-2012 scripts and dehydrated data."
+  - dest: "assets/conll-eval"
+    git:
+      repo: "https://github.com/conll/reference-coreference-scorers"
+      branch: "master"
+      path: ""
+    description: "CoNLL-2012 reference scorer."
+
+workflows:
+  prep:
+    - prep-data
+    - preprocess
+  train:
+    - train-cluster
+    - prep-span-data
+    - train-span-predictor
+    - assemble
+
+  all:
+    # prep
+    - prep-data
+    - preprocess
+    # train
+    - train-cluster
+    - prep-span-data
+    - train-span-predictor
+    - assemble
+
+commands:
+  - name: "prep-data"
+    help: "Rehydrate the data using OntoNotes"
+    script: 
+      # XXX python2 is required for the old scripts
+      - python2 -c "print('Checking for Python 2...')"
```

Comment:
This can hopefully be removed with https://github.com/explosion/conll-2012/pull/1

Reply:
Thanks, removed it.